src-js/webpack/src/videoobject/components/video-modal/index.js

```diff
@@ -0,0 +1,130 @@
+                                    description={__("The date the video was published in IS8601 format", "wordlift")}
+                                    value={video.upload_date}
+                                    onChange={onChangeListener}
+                                    identifier={"upload_date"}/>
+                        <ModalField title={__("CONTENT URL", "wordlift")}
+                                    description={__("A URL pointing to the actual video media file", "wordlift")}
+                                    value={video.content_url}
+                                    onChange={onChangeListener}
+                                    identifier={"content_url"}
+                        />
+                        <ModalField title={__("DURATION", "wordlift")}
+                                    description={__("The duration of the video in ISO 8601 format.", "wordlift")}
+                                    value={video.duration}
+                                    onChange={onChangeListener}
+                                    identifier={"duration"}/>
+                        <ModalField title={__("EMBED URL", "wordlift")}
+                                    description={__("A URL pointing to a player for the specific video.", "wordlift")}
+                                    value={video.embed_url}
+                                    onChange={onChangeListener}
+                                    identifier={"embed_url"}/>
+                    </WlColumn>
+                </WlContainer>
+            </React.Fragment>)
+    }
+
+
+    render() {
+        return (
+            <WlBgModal shouldOpenModal={this.props.isModalOpened} key={this.props.videoIndex}>
+                <WlModal shouldOpenModal={this.props.isModalOpened} className={"wl-video-modal--full-width"}>
+                    {this.renderIfVideoExists()}
+
+                        <WlContainer fullWidth={true}>
+                            <WlColumn className={"wl-col--width-85"}>
+                            </WlColumn>
+                            <WlColumn className={"wl-col--width-15"}>
+                                <WlActionButton className={"wl-action-button--primary"}
+                                                text={__("Save", "wordlift")}
+                                                onClickHandler={() => {
+                                                    this.props.dispatch(saveVideoDataRequest())
+                                                }}/>
+                            </WlColumn>
+                        </WlContainer>
+                </WlModal>
+
+            </WlBgModal>
+        )
+    }
+
+}
+
+const mapStateToProps = (state) => {
+    return {
+        isModalOpened: state.isModalOpened,
+        videoIndex: state.videoIndex,
+        video: state.videos[state.videoIndex],
+        isNextEnabled: state.videoIndex + 1 < state.videos.length,
```

Comment:
@naveen17797 doesn't `state.videoIndex + 1 < state.videos.length` mean `state.videoIndex <= state.videos.length`?

Reply:
No, for example

let a  = [1,2,3]

a.length would be `3`

when i am at index 2 i want to check if the next element exists on index 3

`state.videoIndex <= state.videos.length`

this translates to 

`2 <= 3`

which returns true, but actually it need to return false in order to disable the element

so `2  + 1 < 3`  =>  `state.videoIndex + 1 < state.videos.length`